internal/backends/qt/qt_window.rs

```diff
@@ -1751,6 +1752,22 @@ impl PlatformWindow for QtWindow {
             }};
         }
     }
+
+    fn position(&self) -> euclid::Point2D<f32, PhysicalPx> {
+        let widget_ptr = self.widget_ptr();
+        let qp = cpp! {unsafe [widget_ptr as "QWidget*"] -> qttypes::QPoint as "QPoint" {
+            return widget_ptr->pos();
+        }};
+        euclid::Point2D::new(qp.x as _, qp.y as _)
+    }
+
+    fn set_position(&self, position: euclid::Point2D<f32, PhysicalPx>) {
```

Comment:
The question is whether we want to use physical it logical here. Especially because this might be at a point where the scale factor is unknown

Reply:
Yes, that caused it not to work with the GL backend - I noticed that last night. That's fixed now :)